From: Thomas Weber <tweber@debian.org>
Description: Remove configure.h to enable correct out-of-source builds
 The file is included in the upstream tarball due to the way the release
 tarball is generated.
Forwarded: not-needed

--- a/qtoctave/src/configure.h
+++ /dev/null
@@ -1,11 +0,0 @@
-
-#ifndef __CONFIGURE_H__
-#define __CONFIGURE_H__
-#define CONFIG_PATH "/opt/qtoctave/share/qtoctave"
-#define HELP_PATH "/opt/qtoctave/share/doc/octave-html/octave_doc/index.html"
-#define ICON_PATH "/opt/qtoctave/share/qtoctave/images"
-#define LANG_PATH "/opt/qtoctave/share/qtoctave/lang"
-#define QTOCTAVE_HELP_PATH "/opt/qtoctave/share/doc/qtoctave/qtoctave_doc/index.html"
-
-#define PKG_CMD_PATH "/opt/qtoctave/share/qtoctave/pkg-commands.list"
-#endif
